tlts_detector:
  ros__parameters:
    ts_conf_threshold: 0.61         # Traffic sign confidence threshold
    tl_green_threshold: 0.6         # Traffic light green confidence threshold
    tl_yellow_threshold: 0.2        # Traffic light yellow confidence threshold
    tl_red_threshold: 0.45          # Traffic light red confidence threshold